src/services/resources/topHeadlines/index.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  TopHeadlinesDTO,
+  TopHeadlinesResponseData,
+} from '~/@types/dtos/topHeadlinesDTO';
+import api from '~/services/api';
+
+export const fetchHeadlines = async (
+  payload: TopHeadlinesDTO
+): Promise<TopHeadlinesResponseData> => {
+  const {data} = await api.get<Promise<TopHeadlinesResponseData>>(
+    '/top-headlines',
```

Comment:
Base URL already has the `/` within the URL... there is no need to add it on the endpoint as well

```suggestion
    'top-headlines',
```